jenkins/opensearch/integ-test.jenkinsfile

```diff
@@ -58,56 +54,88 @@ pipeline {
+                            node(AGENT_LABEL) {
+                                withDockerContainer(args: DOCKER_AGENT.args, image: DOCKER_AGENT.image){
+                                    try {
+                                        stage("Run Integtest ${local_component}") {
+                                            echo "Component Name: ${local_component}"
+                                            unstash 'opensearch-build-repo'
+                                            // Jenkins tend to not clean up workspace at times even though ws clean is called
+                                            // Due to docker is mounting the agent directory so it can communicated with the agent
+                                            // This sometimes cause the test to retain last run results and trigger build failures
+                                            sh("rm -rf test-results")
+                                            runIntegTestScript(
+                                                jobName: BUILD_JOB_NAME,
+                                                componentName: "${local_component}",
+                                                buildManifest: BUILD_MANIFEST,
+                                                testManifest: "manifests/${TEST_MANIFEST}"
+                                            )
+                                        }
+                                    } catch (e) {
+                                        echo "Error running integtest for component ${local_component}"
+                                        throw e
+                                    } finally {
+                                        uploadTestResults(
+                                            buildManifestFileName: BUILD_MANIFEST,
+                                            jobName: JOB_NAME,
+                                            buildNumber: BUILD_ID
+                                        )
+                                        postCleanup()
+                                    }
+                                }
+                            }
+                        }
+                    }
+                    parallel component_tests
                 }
             }
             post {
                 always {
-                    script {
-                        uploadTestResults(
-                            buildManifestFileName: BUILD_MANIFEST,
-                            jobName: JOB_NAME,
-                            buildNumber: BUILD_ID
-                        )
-                    }
+                    //script {
+                    //    uploadTestResults(
+                    //        buildManifestFileName: BUILD_MANIFEST,
+                    //        jobName: JOB_NAME,
+                    //        buildNumber: BUILD_ID
+                    //    )
+                    //}
                     postCleanup()
```

Comment:
I think you forget to uncomment these and the message section below.

Reply:
Fixed.